dhis2_android_maps/src/main/java/org/dhis2/maps/usecases/MapStyleConfiguration.kt

```diff
@@ -6,6 +6,9 @@ import org.hisp.dhis.android.core.D2
 import org.hisp.dhis.android.core.map.layer.MapLayerImageryProvider
 
 class MapStyleConfiguration(private val d2: D2) {
+
+    private val canCaptureManually by lazy { true }
```

Comment:
I'm not sure why this is lazy if is not assigned

Reply:
It is to make it ready for ANDROAPP-6230. I've would have added a TODO comment, but sonnarcloud did not like it XD